### ISSUE TITLE
Running `fim-mount` hangs forever when mount directory doesn't exit

### ISSUE THREAD

**Issue by hwittenborn** [ruanformigoni/flatimage#17]
When attempting to run `fim-mount` on the alpine image, I noticed that running `./alpine.fim fim-mount` or `./alpine.fim fim-mount ./dir` (where `./dir` doesn't exist) causes the command to just hang forever with no output.

I'm assuming this is a bug, but I'm not positive.

**Comment by hwittenborn** [comment]
I think this is a bug too, but I ran `mkdir ./mount` first (`./mount` is the directory I'm mounting at), and running `fim-mount ./mount` is showing this message:

```
/tmp/fim/instance/7de4257_20231001041443/boot: line 635:  3183 Segmentation fault      "$FIM_DIR_GLOBAL_BIN"/e2fsck -fy "$FIM_FILE_BINARY"\?offset="$FIM_OFFSET" &> "$FIM_STREAM"
```

**Comment by hwittenborn** [comment]
It looks like the segfault in that last comment was just due to my image, redownloading a new copy of the alpine image removed the segfault.

The new image is showing this error about running `touch` on a file when using a non-existent directory though (and the original error is now gone):

```
touch: cannot touch '/tmp/fim/instance/7de4257_20231001041443/mount/dNcAol/fim/config': No such file or directory
```

**Comment by hwittenborn** [comment]
And now I'm back to the hanging again, I'm not quite sure what triggered the above scenario :p.

**Comment by ruanformigoni** [comment]
Fixed in [v0.2.3](https://github.com/ruanformigoni/flatimage/releases/tag/v0.2.3)
